cmd/ztest/ztest.c

```diff
@@ -6687,7 +6694,11 @@ ztest_deadman_thread(void *arg)
 
 		(void) printf("ztest has been running for %lld seconds\n",
 		    total);
+
+		last_run = gethrtime();
```

Comment:
Once we exceed the initial delay, I think it would be reasonable to set `delay = MSEC2NSEC(zfs_deadman_checktime_ms)`.